components/float-button/FloatButtonContent.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { memo } from 'react';
+import FileTextOutlined from '@ant-design/icons/FileTextOutlined';
+import classNames from 'classnames';
+import type { FloatButtonContentProps } from './interface';
+
+const FloatButtonContent: React.FC<FloatButtonContentProps> = props => {
+  const { icon, description, prefixCls, type, className, ...rest } = props;
+  const defaultElement = (
+    <div className={`${prefixCls}-icon ${prefixCls}-${type}-icon`}>
```

Comment:
```suggestion
    <div className={`${prefixCls}-icon`}>
```
不应该对默认 icon 做处理，应该直接覆盖 anticon 的样式